.github/workflows/check_mergeability_ghstack.yml

```diff
@@ -2,79 +2,35 @@ name: Check mergeability of ghstack PR
 
 on:
   pull_request:
-    types: [opened, synchronize, reopened, edited]
+    types: [opened, synchronize, reopened]
```

Comment:
Might be a naive q, but why do we exclude edited here? Isn't that important for checking mergeability?

Reply:
The doc say https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=edited#pull_request `edited` event is when:

> The title or body of a pull request was edited, or the base branch of a pull request was changed.

The first 2 cases are not needed while the last one is kind of in a gray area.  I guess it's ok because who wants to make thing complicated in ghstack by trying to change the base branch.

Reply:
Not particularly:
>  edited | title, body, or the base branch of the PR is modified
> https://frontside.com/blog/2020-05-26-github-actions-pull_request/

If any actual commits are modified, `synchronize` will trigger.